layouts/partials/fragments/hero.html

```diff
@@ -4,9 +4,13 @@
 
 <header>
   {{ with $hero.header }}
-    <div class="jumbotron text-center header-image mb-0">
+    <div class="jumbotron text-center header-image mb-0
+     {{- printf " bg-%s" $bg -}}
+    ">
   {{ else }}
-    <div class="jumbotron text-center mb-0">
+    <div class="jumbotron text-center mb-0
+     {{- printf " bg-%s" $bg -}}
+    ">
```

Comment:
Not necessary here, but this could be simplified to:
```
<div class="jumbotron text-center mb-0
  {{- printf " bg-%s" $bg -}}
  {{ with $hero.header }}
    {{ printf " header-image" }}
  {{ end }}
">
```

Reply:
Alright sure will update that.

Reply:
It was my code that was unoptimized to beginn with ;)
If you see something similar, you are happy to change it.